he.json

```diff
@@ -0,0 +1,297 @@
+                    "no-data" : "אין נתונים זמינים - נסה שוב בעוד כמה רגעים"
+                }
+            }
+        },
+        "upcoming" : {
+            "title" : "אייטמים צפויים בקרוב",
+            "line1" : "כל האייטמים שעדיין לא פורסמו בפורטנייט באטל רויאל"
+        },
+        "list" : {
+            "title" : "רשימת אייטמים",
+            "line1" : "כל אייטם קוסמטי זמין בפורטנייט באטל רויאל",
+            "table" : {
+                "icon" : "אייקון",
+                "name" : "שם",
+                "type" : "סוג",
+                "rarity" : "נדירות",
+                "obtained" : "להשיג באמצעות..."
+            }
+        },
+        "history" : {
+            "list" : {
+                "title" : "היסטוריית האייטם שופ",
+                "line1" : "להלן 8 הימים האחרונים של סיבובי האייטם שופ עבור פורטנייט באטל רויאל",
+                "line2" : "לחץ על אחת כדי לצפות באייטם שופ המלא, או גלול כדי להציג את היסטורית החנות המלאה לפי יום."
+            },
+            "view" : {
+                "title" : "סבב החנות",
+                "line1" : "זהו סבב האייטם שופ של ה[date] עבור פורטנייט באטל רויאל",
+                "line2" : "לחץ על אייטם קוסמטי כלשהוא כדי לראות מידע נוסף אודותיו.",
+                "today-button" : "האייטם שופ הנוכחי"
+            }
+        },
+        "news" : {
+            "title" : "חדשות",
+            "line1" : "עדכוני החדשות הרשמיים מהמשחק. מתעדכן בכל דקה.",
+            "error" : "לא הצלחנו לקבל את החדשות מהמשחק, אך ננסה שוב בזמן הקרוב.",
+            "more-ingame-button" : "טען עוד"
+        },
+        "randomiser" : {
+            "title" : "סט אקראי",
+            "line1" : "בחר תלבושת אקראית מתוך כל האייטמים הקיימים.",
+            "chooseText" : "לחץ כדי לבחור",
+            "randomise-all" : "הגרל את כולם"
+        },
+        "png" : {
+            "title" : "אייטמים קוסמטיים בפורמט PNG",
+            "line1" : "אנא השתמש בתמונות כאשר אתה נותן קרדיט לאתר fnbr.co. התמונות בתצוגה המקדימה מוקטנות, אך ההורדה שלהם תתבצע בגודל המקורי",
+            "sort-by" : "מיין לפי:",
+            "all" : "הכל"
+        },
+        "gallery" : {
+            "title" : "גלריית האייטמים הקוסמטיים",
+            "line1" : "כל אייטם קוסמטי הזמין בפורטנייט באטל רויאל",
+            "line2" : "אנא השתמש בתמונות כאשר אתה נותן קרדיט לאתר fnbr.co"
+        },
+        "icons" : {
+            "title" : "אייקוני פורטנייט באטל רויאל",
+            "line1" : "כל התמונות בדף זה שייכות לאפיק גיימס."
+        },
+        "item" : {
+            "unreleased" : "לא פורסם",
+            "description" : "תיאור",
+            "history" : {
+                "title" : "היסטוריית החנות",
+                "release-never" : "תאריך שחרור: אף פעם",
+                "release-date" : "תאריך שחרור:",
+                "last-never" : "נראה לאחרונה: אף פעם",
+                "last-date" : "נראה לאחרונה:",
+                "occurrences" : "פעמים באייטם שופ: "
+            },
+            "reminder-count" : "משתמשים עם [תזכורות](/reminders):",
```

Comment:
It's not allowed to separate the link name (within the square brackets `[...]`) from the link url (inside the round brackets `(...)`). It must be in this format: `[link name](link url)`